cscope.py

```diff
@@ -296,12 +293,15 @@ def run_cscope(self, mode, word):
                 sublime.error_message("Cscope ERROR: %s failed!" % cscope_arg_list)
 
         output, erroroutput = proc.communicate()
+
         # print output
         # print erroroutput
-        try:
-            output = str(output, encoding="utf8")
-        except TypeError:
-            output = unicode(str(output), encoding="utf8")
+
+        if not popen_arg_list["universal_newlines"]:
+            try:
+                output = str(output, encoding="utf8")
+            except TypeError:
+                output = unicode(str(output), encoding="utf8")
 
```

Comment:
It also looks like this should be unnecessary, as you stated earlier. Can you just delete lines 296-304? The commented-out prints should be removed and it looks like the try/except can go too?